src/OlStyleParser.ts

```diff
@@ -1007,10 +1013,79 @@ export class OlStyleParser implements StyleParser {
       lineDash: symbolizer.outlineDasharray,
     }) : null;
 
-    return new this.OlStyleConstructor({
-      stroke: stroke,
-      fill: fill
+    const olStyle = new this.OlStyleConstructor({
+      fill,
+      stroke
+    });
+
+    if (symbolizer.graphicFill) {
+      const pattern = this.getOlPatternFromGraphicFill(symbolizer.graphicFill);
+      if (!fill) {
+        fill = new this.OlStyleFillConstructor({});
+      }
+      fill.setColor(pattern);
```

Comment:
If `pattern` is null, this will override whatever fill was using as default (and I think we should leave default fill as fallback).
```suggestion
      if (pattern) {
        fill.setColor(pattern);
      }
```